ios/Wallet/Wallet.swift

```diff
@@ -109,19 +109,19 @@ class Wallet: NSObject {
 extension Wallet: WalletProtocol {
     func onIdentifyWriteSuccess() {
         EventEmitter.sharedInstance.emitNearbyMessage(event: "exchange-receiver-info", data: Self.EXCHANGE_RECEIVER_INFO_DATA)
-        print("wallet delegate called")
+        os_log(.info, "wallet delegate called")
     }
 
     func onDisconnectStatusChange(data: Data?){
-        print("Handling notification for disconnect handle")
+        //print("Handling notification for disconnect handle")
         if let data {
             let connStatusID = Int(data[0])
             if connStatusID == 1 {
-                print("con statusid:", connStatusID)
+                //print("con statusid:", connStatusID)
                 destroyConnection()
             }
         } else {
-            print("weird reason!!")
+            //print("weird reason!!")
```

Comment:
This error condition needed to be printed with proper reason using os_log.

Reply:
Otherwise the error is escaped.

Reply:
Yes. uncommented.